docs/index.md

```diff
@@ -5,28 +5,17 @@
 layout: home
 ---
 
-Two Factor Authentication (2FA) is a fantastic way to 
-make it harder for hackers to hijack your accounts, even 
-if they somehow steal your password.
+<h1 class="nav-header">2FA Notifier</h1>
+
+Two-Factor Authentication (2FA) is a fantastic way to make it harder for hackers to hijack your accounts, even if they somehow steal your password.
 
 **But, it only works if you actually enable it!!**
 
 2FA Notifier let's you know which sites you visit
 support 2FA and specifically how to enable it.
 
-Anytime you visit a site that supports 2FA, a notification 
-will pop up in the top right of your computer. When you click 
-on it, you will be brought directly to the site's documentation
-explaining how to enable 2FA.
-
-**The right information, right when you need it!**
+<a href="https://chrome.google.com/webstore/detail/lggnfpepjfjffimehbjeofegcbmhogoj/publish-accepted?authUser=0&hl=en-US" class="cta-btn">Get 2FA Notifier</a>
```

Comment:
I updated this link on the master branch. Can you make sure you have the latest merged in?